resume.md

```diff
@@ -1,49 +1,65 @@
-# FIRST LAST NAME
+# Armen Phelps
 
-(123) 456 - 7890 | email@gmail.com
+(510) 846 - 6136 | armenphelps204@berkeley.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+https://www.linkedin.com/in/armenphelps2022/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+University of California, Berkeley | B.A. Computer Science, B.A. Applied Math (Double Major) | May 2022
+Coursework: Structure & Interpretation of Computer Programs; Data Structures; Computer Architecture (Machine Structures); Computer Security; Efficient Algorithms & Intractable Problems; Designing Information Devices & Systems; Abstract Algebra; Discrete Math & Probability Theory; Linear Algebra & Differential Equations; Multivariable Calculus
+
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+Languages: Python, NumPy, Java, C, Golang, SQL, RISC-V, x86, Scheme, LaTeX
+Libraries/Frameworks: IntelliJ, Excel, Regex
+Version Control: Git
+
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+Academic Intern: UC Berkeley Computer Science Department | Berkeley, CA | June 2019 - Dec 2019
+* Assisted in labs, office hours, and coding assignments for students in CS61A and CS61B
+* Contributed 100+ hours over 2 semesters
+* Beta tested and gave feedback on a newly designed class project for CS61A about building a spellchecker system
```

Comment:
this sounds like a big project for CS61A - can you dive more into the technical details here?